docs/sources/storage/_index.md

```diff
@@ -179,24 +180,24 @@ When a new schema is released and you want to gain the advantages it provides, y
 
 First, you'll want to create a new [period_config]({{< relref "../configure#period_config" >}}) entry in your [schema_config]({{< relref "../configure#schema_config" >}}). The important thing to remember here is to set this at some point in the _future_ and then roll out the config file changes to Loki. This allows the table manager to create the required table in advance of writes and ensures that existing data isn't queried as if it adheres to the new schema.
 
-As an example, let's say it's 2020-07-14 and we want to start using the `v11` schema on the 20th:
+As an example, let's say it's 2023-07-14 and we want to start using the `v12` schema on the 20th:
```

Comment:
Given this is already deprecated, let's leave the table manager example to use boltdb itself? 
we could just update the period to 24h to ensure correctness.

Reply:
👍  previous paragraph talks about table manager managing the schema. so I thought this was inside the same section.

guess this page needs a restructure. I can pick this in a follow-up